include/grpcpp/impl/codegen/call_op_set.h

```diff
@@ -421,17 +421,14 @@ Status CallOpSendMessage::SendMessagePtr(const M* message) {
 template <class R>
 class CallOpRecvMessage {
  public:
-  CallOpRecvMessage()
-      : got_message(false),
-        message_(nullptr),
-        allow_not_getting_message_(false) {}
+  CallOpRecvMessage() {}
```

Comment:
remove the constructor altogether, or mark it =default ?

Reply:
ah, didn't realize that, will do